rules.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: Community Rules
+permalink: /rules/
+---
+
+##ITC should be considered SFW (Safe for Work). As a rule of thumb, before you post anything, consider if you'd want a co-worker or your boss to see it.
+
+###The following is not allowed:
+
+- Any content which is not in keeping with the Code of Conduct.
+- Blatantly sexual content (pornography, nudity, etc.)
+- Misogyny, ableism, sexism racism, nationalist insults etc.
+- Personal abuse.
+- Links to anything which would break the rules if posted directly.
+- Posting any private phone numbers.
+- Advocating criminality or violence.
+- Trolling (A 'troll' is someone who posts controversial, inflammatory or off-topic messages with the primary intent of provoking other users. These users should be reported to admins immediately.)
+- Posting libellous material.
+- Txt spk (text speak).
+- Misdirection or deceiving links (such as to sites which hijack the browser, rickroll, etc.).
+- Anything illegal.
+- Defame, abuse, harass, stalk, threaten or otherwise violate the rights of others.
+- Identify or speculate as to the identity of any pseudonymous user
+- Soliciting passwords or personally identifying information for commercial or unlawful purposes
+- Posting content in respect of any matter that is currently before the courts
+
+
+### You may not join if:
+
+- You are currently banned for any reason.
+- You are joining solely to spam.
+- You have been criminally convicted.
```

Comment:
Right now I'd say leave it out. There's plenty of criminal convictions I'd imagine we'd have no issue with. It's a sweeping rule that tackles a problem we haven't encountered. 

If the concern is say sex offenders harassing people or sharing shall we say, unsuitable media, then we can ban them based on other rules.